assets/src/styles/blocks.scss

```diff
@@ -32,3 +32,7 @@
 // Other
 @import "blocks/WideBlocks";
 @import "blocks/Editor";
+
+.is-style-rounded {
```

Comment:
I'm not sure about the Quick Links pattern, but for me with the Deep Dive pattern in order to add this [animation](https://www-dev.greenpeace.org/gutenberg/deep-dive-topics-links/) I think I would need the border radius to be applied to the image or figure element, so more something like this:
```
&.is-style-rounded {
    figure,
    img {
      border-radius: 50%;
    }
  }
```
Would that work for your use case too?

Reply:
Also, maybe it would make more sense to define these styles in [Image.scss](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/master/assets/src/styles/blocks/core-overrides/Image.scss)?